### DISCUSSION TITLE
Is Flowbite-Svelte official or unofficial Flowbite component library for Svelte?

### DISCUSSION THREAD

**Discussion by danilodeveloper** [themesberg/flowbite-svelte#104] [General]
Hi there!

I would like to know if Flowbite-Svelte is really an official Flowbite component library for Svelte. I'm asking because in the Flowbite site it says that Flowbite-Svelte is an **unofficial** Flowbite component library for Svelte ([link](https://flowbite.com/docs/getting-started/svelte/)) and the github repo says that it's an official Flowbite component. which one is correct?

Congrats for the work!

tks!
dan

**Comment by shinokada** [comment]
Hi Dan,

Yes, it is an official Flowbite component library for Svelte.
We just need to update the doc.
